### ISSUE TITLE
Installation via bower?

### ISSUE THREAD

**Issue by binarykitchen** [pheuter/require-j#5]
Since this is used on the client-side, why don't you add bower support? Would be cool. Thx!

**Comment by pheuter** [comment]
Yeah, totally, makes sense. I'll try to make some time soon for this.

**Comment by binarykitchen** [comment]
Cool, thanks. Hope that will be soon :)
